source/application.py

```diff
@@ -0,0 +1,18 @@
+from flask import Flask
+from werkzeug.contrib.fixers import ProxyFix
+
+from endpoints.api import api
+from model.Database import Database
+from plugins.simpleLock.simpleLock import simpleLock
+
+
+DAO = Database()
```

Comment:
For this particular instance I already changed it. But for the whole project I made an issue (#20) to  convert it to the PIP8 standard.